### PR TITLE
Fix couchbase url backward compatibility

### PR DESCRIPTION
**Pull request by Yshayy**

### CODE REVIEW

File: addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs [hunk 1]
```diff
@@ -42,7 +42,11 @@ public void Configure(IServiceCollection services, IConfiguration configuration)
             var couchbaseConfig = configuration.GetSection("Couchbase");
             var contextBucketName = couchbaseConfig["BucketName"];
             var contextBucketPassword = couchbaseConfig["Password"];
+            var serverUrl = couchbaseConfig.GetSection("Url").Get<string>();
             var serverUrls = couchbaseConfig.GetSection("Urls").Get<List<string>>();
+            if (serverUrls == null || serverUrls.Count == 0){
+                serverUrls = new List<string>(){ serverUrl };
```
Comment (AleF83) [inline addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs]: Maybe check that Url exits and not empty?

Comment (Yshayy) [reply addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs]: What we can we do if that's the case?

Comment (Yshayy) [reply addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs]: It's more like use the old api if "urls" is not used

Comment (Yshayy) [reply addons/Context/Tweek.Drivers.Context.Couchbase/CouchBaseServiceAddon.cs]: It'll probably get the same exception as before in the ```new Uri(url)``` code

**Pull request merged**